### DISCUSSION TITLE
Get Publisher Display Name

### DISCUSSION THREAD

**Discussion by DavidM29** [unoplatform/uno#18129] [Q&A] [answered]
Hello, I know I can get app display name from `Windows.ApplicationModel.Package` class, but the `PublisherDisplayName` property is not implemented there, is there any other way to get publisher display name (besides from `PackageId`, because it gets the name from the package signing and not the display name)?

Edit: as I checked, `Publisher` property is not implemented in the `PackageId` either.

**Comment by morning4coffe-dev** [comment, marked as answer]
Hi @DavidM29,

It seems that the `PublisherDisplayName` is currently implemented only on Windows and Skia targets with localization support. Please feel free to correct me if I’m mistaken, but as this feature isn’t commonly required, you might need to implement it specifically for your platform. Alternatively, I would suggest adding a static string with the publisher name to your resources. May I ask what your specific use case for this feature is?

**Reply by DavidM29** [reply]
Hello @morning4coffe-dev, thanks for the answer, I went with the const string.
the use case is to create a directory for a local database file.
